doc/whats_new/v0.20.rst

```diff
@@ -120,6 +120,10 @@ Classifiers and regressors
   error for prior list which summed to 1.
   :issue:`10005` by :user:`Gaurav Dhingra <gxyd>`.
 
+- Fixed a bug in :class:`linear_model.OrthogonalMatchingPursuit` that was
```

Comment:
I suppose this should be mentioned in changed models above so that users expect different results without reading the full changelog.